app/routes.py

```diff
@@ -0,0 +1,279 @@
+from app import db 
+from flask import Blueprint, jsonify, request, make_response 
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timedelta
+import requests
+import os
+from dotenv import load_dotenv
+
+customers_bp = Blueprint("customers",__name__,url_prefix = "/customers") # path that gives you access to resources/endpoint
+videos_bp = Blueprint("videos",__name__,url_prefix = "/videos")
+rentals_bp = Blueprint("rentals",__name__,url_prefix = "/rentals")
+
+@videos_bp.route("", methods=["GET", "POST"])
+def handle_videos():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body: 
+            return jsonify({"details":"Request body must include title."}), 400
+        if "release_date" not in request_body: 
+            return jsonify({"details":"Request body must include release_date."}), 400
+        if "total_inventory" not in request_body: 
+            return jsonify({"details":"Request body must include total_inventory."}), 400
```

Comment:
You can move these guard clauses to a helper function and make it dynamic then you could use it in a bunch of different endpoints. You would look something like 
```
def validate_parameters(request_body, parameters):
           for attribute in request_body:
                if attribute not in request_body:
                       abort(make_response({"details": f"Request body must include {attribute}."}, 400))
```